### ISSUE TITLE
non-convergence problem in Recurrence 

### ISSUE THREAD

**Issue by lfrcoder** [CPFLAME/LOCO#4]
Your work is excellent, during the experiment.I have a few questions to ask you.
Why does your training log show solver='models/pascal_voc/VGG16/faster_rcnn_end2end/solver.prototxt' instead of contxt-solver？

**Comment by lfrcoder** [comment]
When I got caffe-fast-rcnn, I used the author's py-caffe-faster version. Because of the version you provided, there is no smooth_L1_loss_layer in layers in caffe, so I use the caffe of the faster, I don't know if this is the reason. Causing an experiment error

**Comment by CPFLAME** [comment]
For question1：
please make sure you have run this command：
![image](https://user-images.githubusercontent.com/13843079/43450702-e3affde6-94e5-11e8-9298-62417a8ffe2d.png)
in this command line it run context-solver
![image](https://user-images.githubusercontent.com/13843079/43450798-1c3732d8-94e6-11e8-8e91-2b22c82c1634.png)

For question2：
I have mentioned it in Installation：
cd LOCO
git clone https://github.com/rbgirshick/caffe-fast-rcnn.git

**Comment by lfrcoder** [comment]
Thank you very much for your answer.！I did use contxt_solver.txt during the experiment. After git clone https://github.com/rbgirshick/caffe-fast-rcnn.git, there is no smoothl1 in caffe's layer.
but its needed to train, so I replaced the caffe of the faster: https://github.com/rbgirshick/caffe-fast-rcnn/tree/0dcd397b29507b8314e252e850518c5695efbb83 (with smoothl1),Otherwise, an error will be reported:No smoothl1 layer,I am confused about this.Looking forward to your reply, thank you!

**Comment by CPFLAME** [comment]
Maybe you can use the Microsoft one:
git clone https://github.com/Microsoft/caffe.git
I guess this could help

**Comment by lfrcoder** [comment]
Thank you ！There is a phenomenon here，I can also train with the caffe of the faster, but the loss is very large and does not converge. I want to know the value of the loss  finally  Also, if caffe in the tutorial doesn't work, which one do you use?.Looking forward to your reply, thank you!

**Comment by CPFLAME** [comment]
I use the R.B.G's  one,
By the way, do you have use the pretrained model ?  If you have used, is the loss of the original py-faster-rcnn  large too?
